app/controllers/api/v1x0.rb

```diff
@@ -1,4 +1,9 @@
 module Api
   module V1x0
+    class RootController < ApplicationController
+      def openapi
+        render :json => Rails.root.join('public', 'catalog', 'v0.1.0', 'openapi.json').read
```

Comment:
*`v1.0.0`

Reply:
@syncrou Should the public/catalog folder be updated to match the `v1,0` version?

Reply:
Will merge the PR now and fix the path as part of https://github.com/ManageIQ/catalog-api/pull/183